webapp/channels/src/components/admin_console/system_users/system_users_list/index.tsx

```diff
@@ -50,6 +51,12 @@ function SystemUsersList(props: Props) {
     const [userReports, setUserReports] = useState<UserReport[]>([]);
     const [loadingState, setLoadingState] = useState<LoadingStates>(LoadingStates.Loading);
 
+    function onError(error: ServerError) {
+        // TODO: Some kind of error handling for actions
```

Comment:
The old modals would allow for an error to be pushed, perhaps if the action fails and then the user list would handle it. How do we want to handle this with the new user list?